docs/faqs/faqs.md

```diff
@@ -0,0 +1,212 @@
+
+</details>
+
+
+<details>
+  <summary>Do I need to be an expert developer to work with Amplication?</summary>
+  
+ Amplication is a great tool for developers, whatever their experience.  Even though beginners can get up-and-running quickly with Amplication, you own the code and so can make full use of your code-writing talent to customize yor output and work your magic. 
+  
+</details>
+
+<details>
+  <summary>How do I get started with Amplication</summary>
+  
+ You can generate your first app in minutes. These instructions will walk you through the steps of creating an application, adding an entity, adding roles, and setting permissions on entities:
+
+
+ 
+ [Your First Application](https://docs.amplication.com/docs/first-app/)
+
+  
+</details>
+
+<details>
+  <summary>Is Amplication Free?</summary>
+  
+ The Community Edition (CE) is free and always will be. However, to ensure that we can continue to develop Amplication, the Enterprise Edition (EE) will provide enterprise functionality that may require payment. For more information about the CE and EE, read:
+
+ 
+ [Amplication’s licensing model](https://docs.amplication.com/docs/about/about/licensing/)
+
+  
+</details>
+
+<details>
+  <summary>What is the difference between the Community Edition and the Enterprise Edition?</summary>
+
+  
+  **Community Edition (CE)** – contains Amplication's core functionality. It is open-source and free to use for the whole community. The Amplication Community Edition continues to be licensed under the [Apache 2.0 license](https://github.com/amplication/amplication/blob/master/LICENSE).
+
+  **Enterprise Edition (EE)** - adds advanced operations, security, and commercially orientated features. The Enterprise Edition is built on top of the Community Edition and the additional functionality will be supplied as source-available. While users can view and download the code, ownership of the code remains with Amplication, meaning users cannot resell it. Use of the Amplication Enterprise Edition is subject to the Amplication [Terms of Service](https://amplication.com/terms)
+
+  
+</details>
+
+
+<details>
+  <summary>Who owns the code I generate with Amplication?</summary>
+  
+You own the code.  Develop your app however you wish, and deploy it wherever want; on public cloud, private cloud, or on-premise. 
+  
+</details>
+
+
+<details>
+  <summary>How can I get the code?</summary>
+  
+You can download the source code of your application or push a new Pull Request to a GitHub repository. There is also a code-view option, that enables you to preview the code before it is generated. 
+  
+  
+</details>
+
+
+<details>
+  <summary>What server-side technologies does Amplication use?</summary>
+  
+NestJS, Prisma, PostgreSQL, Passport, GraphQL, Swagger UI, Jest, Docker
+  
+</details>
+
+
+<details>
+  <summary>What client-side technologies does Amplication use?</summary>
+  
+ReactJS, React-Admin, Axios, Formik 
```

Comment:
Amplication focuses on creating the best backend - we only provide the client as a starting point for simple CRUD operations. The user is expected to build their own client